fbw-common/src/systems/instruments/src/ND/shared/map/PseudoWaypointLayer.ts

```diff
@@ -91,16 +94,17 @@ export class PseudoWaypointLayer implements MapLayer<NdSymbol> {
             (this.groundSpeed.value * (Date.now() - this.lastUpdateTime)) / 1000 / 60 / 60) *
           mapParameters.nmToPx;
         const rotate = MathUtils.diffAngle(this.headingWord.get().value, this.trackWord.get().value);
-        context.translate(384, 620);
+        context.translate(384, 378 + mapParameters.centerYBias); // bug for ROSE NAV modes, didn't consider centerYBias
```

Comment:
Very nice bug to have fixed actually 👍 However, I think you can remove the comment as it will not be clear which you are referring to later on.